config.py

```diff
@@ -186,273 +181,323 @@
+#   can be used for resampling raw data. 1 means no decimation.
+#
+# Good Practice / Advice
+# ~~~~~~~~~~~~~~~~~~~~~~
+# Decimation requires to lowpass filtered the data to avoid aliasing.
+# Note that using decimation is much faster than resampling.
+#
+# Example
+# ~~~~~~~
+# >>> decim = 1  # no decimation
+# or
+# >>> decim = 4  # decimate by 4 ie devide sampling frequency by 4
+
+decim = 1
 
 ###############################################################################
 # AUTOMATIC REJECTION OF ARTIFACTS
 # --------------------------------
-# for 04-make_epochs.py
-# 
-
+#
+# Good Practice / Advice
+# ~~~~~~~~~~~~~~~~~~~~~~
+# Have a look at your raw data and train yourself to detect a blink, a heart
+# beat and an eye movement.
+# You can do a quick average of blink data and check what the amplitude looks
+# like.
+#
 #  ``reject`` : dict | None
 #    The rejection limits to make some epochs as bads.
 #    This allows to remove strong transient artifacts.
 #    If you want to reject and retrieve blinks later, e.g. with ICA,
 #    don't specify a value for the eog channel (see examples below).
 #    Make sure to include values for eeg if you have EEG data
-
-reject = {'grad': 4000e-13, 'mag': 4e-12}
-
+#
 # Note
 # ~~~~
 # These numbers tend to vary between subjects.. You might want to consider
 # using the autoreject method by Jas et al. 2018.
 # See https://autoreject.github.io
 #
-# Examples
-# ~~~~~~~~
+# Example
+# ~~~~~~~
 # >>> reject = {'grad': 4000e-13, 'mag': 4e-12, 'eog': 150e-6}
 # >>> reject = {'grad': 4000e-13, 'mag': 4e-12, 'eeg': 200e-6}
 # >>> reject = None
 
-# Good Practice / Advice
-# ~~~~~~~~~~~~~~~~~~~~~~
-# Have a look at your raw data and train yourself to detect a blink, a heart
-# beat and an eye movement.
-# You can do a quick average of blink data and check what the amplitude looks
-# like.
-
+reject = {'grad': 4000e-13, 'mag': 4e-12}
 
 ###############################################################################
 # EPOCHING
 # --------
-# for 04-make_epochs.py
 #
 # ``tmin``: float
 #    A float in seconds that gives the start time before event of an epoch.
+#
+# Example
+# ~~~~~~~
+# >>> tmin = -0.2  # take 200ms before event onset.
 
 tmin = -0.6
 
 # ``tmax``: float
 #    A float in seconds that gives the end time before event of an epoch.
+#
+# Example
+# ~~~~~~~
+# >>> tmax = 0.5  # take 500ms after event onset.
 
 tmax = 1.5
 
 # ``trigger_time_shift`` : float | None
 #    If float it specifies the offset for the trigger and the stimulus
 #    (in seconds). You need to measure this value for your specific
 #    experiment/setup.
+#
+# Example
+# ~~~~~~~
+# >>> trigger_time_shift = 0  # don't apply any offset
 
 trigger_time_shift = -0.0416
 
 # ``baseline`` : tuple
 #    It specifies how to baseline the epochs; if None, no baseline is applied.
+#
+# Example
+# ~~~~~~~
+# >>> baseline = (None, 0)  # baseline between tmin and 0
 
-baseline = (-.6, -.5)  # (None, 0.)
```

Comment:
@agramfort `baseline = (-.6, -.5)`
is intended for the localizer data, because there is an event 500ms prior to the time-locking event, so we want to take a baseline before that. 
Sshould be explained in a comment.